src/coreclr/src/gc/gc.cpp

```diff
@@ -4550,24 +4695,23 @@ void destroy_initial_memory()
     {
         if (memory_details.allocation_pattern == initial_memory_details::ALLATONCE)
         {
-            virtual_free(memory_details.initial_memory[0].memory_base,
-                memory_details.block_count*(memory_details.block_size_normal +
-                memory_details.block_size_large));
+            virtual_free (memory_details.initial_memory[0].memory_base,
+                memory_details.block_count * (memory_details.block_size_normal +
+                    memory_details.block_size_large));
         }
```

Comment:
you'd also want to add the pinned seg size here if you want to be consistent...even though this does not actually get called. 

nit - it seems a little odd that you made some formatting changes but only for part of this method...

Reply:
Ah, this was existing code that was missing pinned seg size. I put it in.

Some of the formatting changes were by mistake, only the space after virtual_free was intentional.